docs/tutorials/08_property_framework.ipynb

```diff
@@ -0,0 +1,1469 @@
+  },
+  {
+   "cell_type": "markdown",
+   "id": "87bf5b74",
+   "metadata": {},
+   "source": [
+    "#### ElectronicIntegrals"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "3807ffbb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit_nature.properties.second_quantization.electronic.integrals import (\n",
+    "    ElectronicIntegrals,\n",
+    "    OneBodyElectronicIntegrals,\n",
+    "    TwoBodyElectronicIntegrals,\n",
+    "    IntegralProperty,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7e2b1630",
+   "metadata": {},
+   "source": [
+    "The `ElectronicIntegrals` are a container class for _n-body_ interactions in a given basis, which can be any of the following:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "1e969daa",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit_nature.properties.second_quantization.electronic.bases import ElectronicBasis"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "b3ead61a",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[<ElectronicBasis.AO: 'atomic'>,\n",
+       " <ElectronicBasis.MO: 'molecular'>,\n",
+       " <ElectronicBasis.SO: 'spin'>]"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "list(ElectronicBasis)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "37aae51a",
+   "metadata": {},
+   "source": [
+    "Let us take the `OneBodyElectronicIntegrals` as an example. As the name suggests, this container is for 1-body interaction integrals. You can construct an instance of it like so:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "55fc757d",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "(MO) 1-Body Terms:\n",
+      "\tAlpha\n",
+      "\t[0, 0] = 1.0\n",
+      "\t[1, 1] = 1.0\n",
+      "\tBeta\n",
+      "\t[0, 0] = 2.0\n",
+      "\t[1, 1] = 2.0\n"
+     ]
+    }
+   ],
+   "source": [
+    "one_body_ints = OneBodyElectronicIntegrals(\n",
+    "    ElectronicBasis.MO,\n",
+    "    (\n",
+    "        np.eye(2),\n",
+    "        2 * np.eye(2),\n",
+    "    ),\n",
+    ")\n",
+    "print(one_body_ints)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c7c1b39e",
+   "metadata": {},
+   "source": [
+    "As you can see, the first argument simply specifies the basis of the integrals.\n",
+    "The second argument requires a little further explanation:\n",
+    "\n",
+    "1. In the case of the `AO` or `MO` basis, this argument **must** be a pair of numpy arrays, where the first and second one are the alpha- and beta-spin integrals, respectively.\n",
+    "\n",
+    "**Note:** The second argument may be `None`, in which case the alpha-spin integrals are used instead.\n",
```

Comment:
```suggestion
    "**Note:** The second argument may be `None`, for the case where the beta-spin integrals are the same as the alpha-spin integrals, so there is not need to provide the same values twice.\n",
```
I know the intent of the text, wondering if the above is a better way of saying the intent etc.